app/assets/stylesheets/scaffold.css

```diff
@@ -45,3 +45,7 @@ div.actions {
 label {
   display: block;
 }
+
+label.required:after {
+  content:"*" 
```

Comment:
Consider adding a whitespace between `content:` and `"*"` for consistency reasons (see e. g. the [relevant entry in the Google HTML/CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops)).